package.json

```diff
@@ -35,45 +35,45 @@
   },
   "devDependencies": {
     "@gravitational/build": "workspace:*",
-    "@storybook/addon-toolbars": "^8.3.4",
-    "@storybook/components": "^8.3.4",
-    "@storybook/react": "^8.3.4",
-    "@storybook/react-vite": "^8.3.4",
+    "@storybook/addon-toolbars": "^8.4.0",
+    "@storybook/components": "^8.4.0",
+    "@storybook/react": "^8.4.0",
+    "@storybook/react-vite": "^8.4.0",
     "@storybook/test-runner": "^0.19.1",
-    "@testing-library/jest-dom": "^6.5.0",
+    "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.0",
     "@testing-library/user-event": "^14.5.2",
-    "@types/jest": "^29.5.13",
-    "@types/node": "^20.16.10",
-    "@types/react": "^18.3.10",
-    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "@types/node": "^20.17.5",
+    "@types/react": "^18.3.12",
+    "@types/react-dom": "^18.3.1",
     "@types/react-highlight": "^0.12.8",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.1.1",
     "@types/react-transition-group": "^4.4.11",
     "@types/wicg-file-system-access": "^2023.10.5",
     "jest": "^29.7.0",
     "jsdom-testing-mocks": "^1.13.1",
-    "msw": "^2.4.9",
+    "msw": "^2.6.0",
```

Comment:
Some tests are failing because of the msw update, here's [an example](https://github.com/gravitational/teleport/actions/runs/11664459496/job/32475027347?pr=48268#step:7:652). This is because of some standard APIs that msw uses that are not available in JSDOM (see https://github.com/mswjs/msw/issues/2340).

It turns out that msw maintains [jest-fixed-jsdom](https://github.com/mswjs/jest-fixed-jsdom). I'm tempted to install it to replace that custom Jest env that we've been using.

Reply:
Unfortunately, it can't be used because it breaks other tests. On top of that, it doesn't have `matchMedia` and `scrollIntoView`, probably because it's focused on APIs actually available in Node.js.

I tried extending it, but that didn't help either. For now I just added `BroadcastChannel` and `TransformStream` manually.